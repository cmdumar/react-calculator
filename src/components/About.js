import React from 'react';

const About = () => (
  <div className="homepage">
    <h1>About</h1>
    <p>This is a practice site for React Router.</p>
  </div>
);

export default About;
