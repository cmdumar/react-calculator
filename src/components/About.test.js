import React from 'react';
import renderer from 'react-test-renderer';
import About from './About';

test('About Page', () => {
  const tree = renderer.create(<About />).toJSON();
  expect(tree).toMatchSnapshot();
});
