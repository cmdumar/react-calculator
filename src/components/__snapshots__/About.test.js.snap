// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About Page 1`] = `
<div
  className="homepage"
>
  <p>
    Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding. â€“William Paul Thurston
  </p>
</div>
`;
